src/AppBundle/Resources/translations/odr-submitted.en.yml

```diff
@@ -10,15 +10,15 @@ whatHappensNext:
     When it's time for you to complete the next report, we will send you a reminder.
     You can make a start on it at any time.
   body2: |
-    If you have any questions about this report, contact the Helpline.
+    If you have any questions about this report, contact the helpline on 0115 934 2700.
+.
```

Comment:
OK I think Simon Busch's point was that we should quote the helpline number. Can you make it so it reads:
If you have any questions about this report, contact the helpline on 0115 934 2700
?